c_api/src/c_api.rs

```diff
@@ -284,6 +308,45 @@ pub unsafe extern "C" fn c2pa_load_settings(
     ok_or_return_int!(result, |_| 0) // returns 0 on success
 }
 
+/// Returns the locations a C2PA manifest was found within the stream.
+/// * C2paManifestLocation::EMBEDDED (0x01)
+/// * C2paManifestLocation::REMOTE (0x02)
+///
+/// This function DOES NOT validate or fetch the C2PA manifest and will never
+/// return `C2paManifestLocation_EMBEDDED`, detection of sidecars depends on
+/// implementation.
+///
+/// # Errors
+/// Returns -1 if there were errors, otherwise returns bit flags corresponding
+/// to the manifest locations. The error string can be retrieved by calling
+/// c2pa_error.
+///
+/// # Safety
+/// Reads from NULL-terminated C strings.
+#[no_mangle]
+pub unsafe extern "C" fn manifest_locations_from_stream(
+    format: *const c_char,
+    stream: *mut C2paStream,
+) -> c_int {
+    let format = CStr::from_ptr(format).to_string_lossy().into_owned();
```

Comment:
Why not use the macro `from_cstr_or_return_null`, so you don't need to import the type just for that call? Like here, seems the exact same case: https://github.com/contentauth/c2pa-rs/blob/e20691aed676c7b827ed177a91cddf66f50cf78e/c_api/src/c_api.rs#L306?